### ISSUE TITLE
Repo, and structure are very missleadin about what snaptier is.

### ISSUE THREAD

**Issue by YashRE42** [snaptier/branding#1]
Snaptier claims that:
`Snaptier automates repetitive development tasks through automated code changes so you can focus on what matters.`
I think this is highly misleading because as far as I've been able to see, this whole project is nothing but init. I don't think it's right to give off the impression that this is something it is not. https://www.snaptier.co/ redirects to a blog titled ![iGossiP](https://user-images.githubusercontent.com/33805964/58755646-a02ac900-8505-11e9-98c1-71c3e7310faf.png).

Please mention in text in your about what the current state of the project its.

**Comment by shroudedcode** [comment]
Sorry for the confusion, @YashRE42. I'll try and address your concerns:

> [...] as far as I've been able to see, this whole project is nothing but init

Regarding our public repositories, yes. But most of the work on this project has been done on a private repository which contains a lot more code that just "init". Still, we've decided to abandon the project for a multitude of reasons (see the [post-mortem write-up](https://user-images.githubusercontent.com/29176678/58759378-3d4b2900-8529-11e9-8da2-6e598cfa7446.png) we shared with the members of our Discord server).

> https://www.snaptier.co redirects to a blog titled iGossiP

That's unfortunate but out of our control. We've let the domain expire and they (whoever that is) bought it.

> Please mention in text in your about what the current state of the project its.

We'll try and make this more clear by making our public repositories private and updating the description of the organization.
